Source/RapyutaSimulationPlugins/Private/Core/RRGameMode.cpp

```diff
@@ -17,61 +17,63 @@
 
 ARRGameMode::ARRGameMode()
 {
+#if RAPYUTA_USE_SCENE_DIRECTOR
     GameStateClass = ARRGameState::StaticClass();
     PlayerControllerClass = ARRPlayerController::StaticClass();
+#endif
 }
 
 void ARRGameMode::PreInitializeComponents()
 {
 #if RAPYUTA_SIM_DEBUG
-    UE_LOG_WITH_INFO(LogRapyutaCore, Warning, TEXT("PreInitializeComponents()"))
+    UE_LOG_WITH_INFO(LogRapyutaCore, Verbose, TEXT("PreInitializeComponents()"))
 #endif
     Super::PreInitializeComponents();
 }
 
 void ARRGameMode::InitGameState()
 {
 #if RAPYUTA_SIM_DEBUG
-    UE_LOG_WITH_INFO(LogRapyutaCore, Warning, TEXT("InitGameState()"));
+    UE_LOG_WITH_INFO(LogRapyutaCore, Verbose, TEXT("InitGameState()"));
 #endif
     Super::InitGameState();
 }
 
 void ARRGameMode::PrintSimConfig() const
 {
-    UE_LOG_WITH_INFO(LogRapyutaCore, Display, TEXT("SIM GAME MODE CONFIG -----------------------------"));
-    UE_LOG_WITH_INFO(LogRapyutaCore, Display, TEXT("- bBenchmark: %d"), bBenchmark);
+    UE_LOG_WITH_INFO(LogRapyutaCore, Verbose, TEXT("SIM GAME MODE CONFIG -----------------------------"));
+    UE_LOG_WITH_INFO(LogRapyutaCore, Verbose, TEXT("- bBenchmark: %d"), bBenchmark);
 }
 
 void ARRGameMode::PrintUEPreprocessors() const
```

Comment:
This function as its name is meant to print useful logs about special UE-builtin settings generally. So if for some reason, it should be turned off by the caller upon certain condition, instead of `Verbose` them all.